addons/sys_modes/fnc_sc_muting.sqf

```diff
@@ -16,18 +16,15 @@
  * Public: No
  */
 
-params ["_radioId1","_radioId2"];
+params ["_radioId1", "_radioId2"];
 
 private _radioTxData = [_radioId1, "getCurrentChannelData"] call EFUNC(sys_data,dataEvent);
 private _radioRxData = [_radioId2, "getCurrentChannelData"] call EFUNC(sys_data,dataEvent);
 
 private _mode1 = HASH_GET(_radioTxData, "mode");
 private _mode2 = HASH_GET(_radioRxData, "mode");
 private _match = false;
-if (
-    (_mode1 == "singleChannel" && _mode2 == "singleChannel") ||
-    (_mode1 == "singleChannelPRR" && _mode2 == "singleChannelPRR")
-) then {
+if ((_mode1 == "singleChannel" && {_mode2 == "singleChannel"}) || {_mode1 == "singleChannelPRR" && {_mode2 == "singleChannelPRR"}}) then {
```

Comment:
Save to 2 variables and compare those.

Reply:
it will result at the end in the same amount of comparisons.

is mode == signleChannelPRR and mode1 == mode2
and mode == singleChannel and mode1 == mode2

Reply:
Will be more readable, that line is very long.